test/utils/url.test.js

```diff
@@ -0,0 +1,72 @@
+import { appendQuery } from 'src/utils';
+import nock from 'nock';
+import expect from 'expect';
+
+/**
+ * @private
+ * Adapted from https://github.com/lakenen/node-append-query
+ */
+describe('appendQuery', function(){
+    it('should append querystring to queryless url', function () {    
+        var result = appendQuery('http://example.com/foo', 'bar=baz&beep=boop')
+            , expected = 'http://example.com/foo?bar=baz&beep=boop'
```

Comment:
Pretty sure style doc has something to say about this :P

Noticed it while writing tests for LiveService today - why are we using es6 imports, but no other es6 code in tests? Arrow funcs won't let you do things like this.timeout(X), but that's rare and I think we should. I think all platforms support these features nowadays, but if we need to use babel for tests, I take this back :)